internal/provider/resource_azure_req_express_dedicated_conn_test.go

```diff
@@ -0,0 +1,42 @@
+package provider
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/PacketFabric/terraform-provider-packetfabric/internal/testutil"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccAzureDedicatedConnectionRequiredFields(t *testing.T) {
+
+	testutil.SkipIfEnvNotSet(t)
+
+	csAzureDedicatedConnectionResult := testutil.RHclCsAzureDedicatedConnection()
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			testutil.PreCheck(t, []string{
+				testutil.PF_ACCOUNT_ID_KEY,
+			})
+		},
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config:                    csAzureDedicatedConnectionResult.Hcl,
+				ExpectNonEmptyPlan:        true,
```

Comment:
As discussed in other PR, let's make sure we don't skip the terraform plan test. so removing `ExpectNonEmptyPlan`. Thanks!